vllm/model_executor/models/llama_eagle3.py

```diff
@@ -245,4 +248,10 @@ def load_weights(self, weights: Iterable[tuple[str, torch.Tensor]]):
                 name = "model." + name
             model_weights[name] = loaded_weight
 
-        return loader.load_weights(model_weights.items())
+        loaded_weights = loader.load_weights(model_weights.items())
+
+        if 'd2t' not in loaded_weights:
+            del self.draft_id_to_target_id
+            self.draft_id_to_target_id = None
```

Comment:
```suggestion
            self.draft_id_to_target_id = None
```

Reply:
why `del` here?

Reply:
I thought it might clear the GPU memory immediately, but that seems not to be the case. Fixed.